backend/dataall/api/Objects/Environment/resolvers.py

```diff
@@ -699,7 +699,51 @@ def get_pivot_role_template(context: Context, source, organizationUri=None):
             )
             raise e
 
+def get_cdk_exec_policy_template(context: Context, source, organizationUri=None):
+    from ....utils import Parameter
 
+    with context.engine.scoped_session() as session:
+        ResourcePolicy.check_user_resource_permission(
+            session=session,
+            username=context.username,
+            groups=context.groups,
+            resource_uri=organizationUri,
+            permission_name=permissions.GET_ORGANIZATION,
+        )
+        cdk_exec_policy_bucket = Parameter().get_parameter(
+            env=os.getenv('envname', 'local'), path='s3/resources_bucket_name'
+        )
+        cdk_exec_policy_bucket_key = Parameter().get_parameter(
+            env=os.getenv('envname', 'local'), path='s3/cdk_exec_policy_prefix'
+        )
+        if not cdk_exec_policy_bucket or not cdk_exec_policy_bucket_key:
+            raise exceptions.AWSResourceNotFound(
+                action='GET_CDK_EXEC_POLICY_TEMPLATE',
+                message='CDK Exec Yaml template file could not be found on Amazon S3 bucket',
+            )
```

Comment:
I think this exception should be added in line 742, as it is the moment in which we actually check if the file exists and we can download it

Reply:
Not applicable, separate PR